### DISCUSSION TITLE
FluentValidation in DataGrid examples?

### DISCUSSION THREAD

**Discussion by jmoerdyk** [Megabit/Blazorise#4750] [Q&A] [answered]
Is using FluentValidation with DataGrid possible?  I can't find any concrete documentation/examples on how to do it.   I have FluentValidation validators working properly with standard forms, but I can't figure out how to wire it up to a DataGrid.

Or is the combination of the two still not possible yet?

**Comment by stsrki** [comment, marked as answer]
DataGrid is still missing FluentValidation support. At the moment it only supports

- Validation handlers
- Pattern validation
- Data annotations

**Comment by jmoerdyk** [comment]
So is the approach in https://github.com/Megabit/Blazorise/issues/1971 still a viable alternative?

**Reply by stsrki** [reply]
It is not because we need first to expose an API on the DatagridColumn that will be able to define HandlerType for the validation. Without it it will always fallback to default handlers that DataGrid is managing internally.
